zcash_client_sqlite/src/wallet.rs

```diff
@@ -1516,51 +1506,36 @@ pub(crate) fn get_account<C: Borrow<rusqlite::Connection>, P: Parameters>(
     let row = result.next()?;
     match row {
         Some(row) => {
-            let account_type: AccountType =
-                row.get::<_, u32>("account_type")?.try_into().map_err(|_| {
-                    SqliteClientError::CorruptedData("Unrecognized account_type".to_string())
-                })?;
+            let kind = parse_account_kind(
+                row.get("account_type")?,
```

Comment:
That was introduced in #1235 and I don't quite get how it is implemented (or when support was added). It might be nice to do a pass over the crate to use that instead of column indexing, if it is not inefficient, but for now I've stuck with whatever style the local code used in each place.